protobuf/options.proto

```diff
@@ -26,25 +26,28 @@ package grakn.protocol;
 // https://github.com/protocolbuffers/protobuf/issues/1606#issuecomment-618687169
 
 message Options {
-    oneof infer_opt {
-        bool infer = 1;
+    oneof inference_opt {
```

Comment:
Is there a reason we rename this from `infer` to `inference`? It now looks inconsistent with our other options (explain, prefetch, read_any_replica...)

Reply:
I see the issue you take with it, will change it to `infer` and `inferLog`